src/Assignment5/MathApp/task5.MathApp.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
```

Comment:
Output Type should be of type ClassLibrary

Reply:
OutputType is changed as ClassLibrary.